languages/kotlin/reference/README.md

```diff
@@ -0,0 +1,50 @@
+# Kotlin reference
+
+## Concepts
+
+Kotlin concept exercises are based on concepts. They are (mostly) connected with the corresponding parts of [official Kotlin reference](https://kotlinlang.org/docs/reference/basic-syntax.html#defining-packages).
+
+### Unsorted
+
+- [Packages](https://kotlinlang.org/docs/reference/basic-syntax.html#defining-packages) (Unpacked)
+- [Entry point](https://kotlinlang.org/docs/reference/basic-syntax.html#program-entry-point) (Unpacked)
+- [Functions](https://kotlinlang.org/docs/reference/basic-syntax.html#defining-functions) (Unpacked)
+- [Variables](https://kotlinlang.org/docs/reference/basic-syntax.html#defining-variables) (Unpacked)
+- [Comments](https://kotlinlang.org/docs/reference/basic-syntax.html#comments) (Unpacked)
+- [String templates](https://kotlinlang.org/docs/reference/basic-syntax.html#using-string-templates) (Unpacked)
+- [Conditional expressions](https://kotlinlang.org/docs/reference/basic-syntax.html#using-conditional-expressions) (Unpacked)
+- [Nullable values and null checks](https://kotlinlang.org/docs/reference/basic-syntax.html#using-nullable-values-and-checking-for-null) (Unpacked)
+- [Type checks and automatic casts](https://kotlinlang.org/docs/reference/basic-syntax.html#using-type-checks-and-automatic-casts) (Unpacked)
+- [for loop](https://kotlinlang.org/docs/reference/basic-syntax.html#using-a-for-loop) (Unpacked)
+- [while loop](https://kotlinlang.org/docs/reference/basic-syntax.html#using-a-while-loop) (Unpacked)
+- [when expression](https://kotlinlang.org/docs/reference/basic-syntax.html#using-a-while-loop) (Unpacked)
+- [Ranges](https://kotlinlang.org/docs/reference/basic-syntax.html#using-a-while-loop) (Unpacked)
+- [Collections](https://kotlinlang.org/docs/reference/basic-syntax.html#using-collections) (Unpacked)
+- [Creating basic classes and their instances](https://kotlinlang.org/docs/reference/basic-syntax.html#using-collections) (Unpacked)
```

Comment:
And here too. Should it be https://kotlinlang.org/docs/reference/basic-syntax.html#creating-basic-classes-and-their-instances ?

Reply:
Yup. Will remove this section as it's only a high-level plan for the document.